tests/test_api.py

```diff
@@ -44,17 +42,31 @@ def test_create_manifestation(client, user):
         'copyright': {
             '@context': '<coalaip placeholder>',
             '@type': 'Copyright',
-            '@id': '',
         },
     }
     resp = client.post(url_for('manifestation_views.manifestationapi'),
                        data=json.dumps(payload),
                        headers={'Content-Type': 'application/json'})
     resp_dict = resp.json
-    assert bool(resp_dict['copyright']['rightsOf']) is True
-    assert bool(resp_dict['manifestation']['manifestationOfWork']) is True
+    copyright_ = resp_dict['copyright']
+    manifestation = resp_dict['manifestation']
+    work = resp_dict['work']
+
+    assert bool(copyright_['rightsOf']) is True
+    assert bool(manifestation['manifestationOfWork']) is True
+
+    # Check @ids
+    assert copyright_['@id'].startswith('../right/')
```

Comment:
I thought about this.
Yes, we should be linking to the asset but throught the transaction id, as it would otherwise become ambiguous in which state the entity currently is.

Additionally, a transaction-based model is commonly used in blockchains, whereas the kinds of Assets we implement are very BigchainDB specific.

Here is my recommendation:

- We link to the asset, but relatively from the transaction so that we get the best of both worlds and have a fall back strategy for blockchains that do not support assets.